packages/drivers/odsp-driver/src/test/epochTests.spec.ts

```diff
@@ -55,19 +54,6 @@ describe("Tests for Epoch Tracker", () => {
 		await epochTracker.removeEntries().catch(() => {});
 	});
 
-	it("defaultCacheExpiryTimeoutMs <= maximumCacheDurationMs policy", () => {
```

Comment:
this test is unnecessary now that we hook into the FiveDays  type which gives us compile time validation of the value